src/roles/roles.go

```diff
@@ -0,0 +1,14 @@
+package roles
+
+// Roles lists the roles
```

Comment:
This comment should be a little clearer as to its intent: it's a list of all valid roles, or it's our full list of known role we support, or something like that.  Nitpicky, I know, but I want our code to be better than my first stab at this app.